lib/ansible/modules/packaging/os/yum.py

```diff
@@ -55,7 +55,7 @@
       - C(present) and C(installed) will simply ensure that a desired package is installed.
       - C(latest) will update the specified package if it's not of the latest available version.
       - C(absent) and C(removed) will remove the specified package.
-      - Default is C(None), however in effect the default action is C(present) unless the C(autoremove) option is¬
+      - Default is C(None), however in effect the default action is C(present) unless the C(autoremove) option is 
```

Comment:
```suggestion
      - Default is C(None), however in effect the default action is C(present) unless the C(autoremove) option is
```

Reply:
thanks

Reply:
@46gorilla thanks for catching that typo!

If you click "Accept suggestion" GitHub will include @samccann's whitespace fix and re-run the test suite. The test suite should pass once the extra whitespace is gone. Once the test suite passes, we can merge this fix.

Reply:
thanks for the advice.
I didnt now how modify my PR , because it is first PR in my life.
Just now , I clicked  "Accept suggestion".

Reply:
W00t! Congratulations on getting your first PR passing our test suite @46gorilla!